emails/weekly-summary.tsx

```
@@ -12,229 +12,256 @@ import {
+        <Text style={digestSubheader}>Week of {weekOf}</Text>
         
-        {/* AI-Generated Welcome Message */}
-        <Section style={aiSection}>
-          <Text style={aiText}>{aiContent.welcomeMessage}</Text>
-          <Text style={aiText}>{aiContent.weeklyInsight}</Text>
-        </Section>
+        <Text style={greeting}>Hey neighbors! 👋</Text>
+        
+        {/* THE WEEK IN REVIEW section */}
+        <Text style={sectionTitle}>THE WEEK IN REVIEW</Text>
+        <Text style={paragraph} dangerouslySetInnerHTML={{ __html: aiContent.weekInReview }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)